blocks/article-grid/article-grid.js

```diff
@@ -0,0 +1,364 @@
+import { getLibs, buildBlock } from '../../scripts/utils.js';
+import initBanner from '../banner/banner.js';
+
+const miloLibs = getLibs();
+const defaultLimit = 9;
+const ROOT_MARGIN = 20;
+
+// Used for getting localized text
+const replacePlaceholder = async (key) => {
+  const { replaceKey } = await import(`${miloLibs}/features/placeholders.js`);
+  const { getConfig } = await import(`${miloLibs}/utils/utils.js`);
+  const result = await replaceKey(key, getConfig());
+
+  return result;
+};
+
+// get data
+function getMetadataTags(doc) {
+  const metaTagElements = doc.head.querySelectorAll('meta[property="article:tag"]');
+  const tagContents = Array.from(metaTagElements, (metaTag) => metaTag.getAttribute('content')).filter(Boolean);
```

Comment:
Don't have to change, but my preference is to use array spread.  Kinda funny it's the exact same amount of chars.
```suggestion
const tagContents = [...metaTagElements].map(metaTag => metaTag.getAttribute('content')).filter(Boolean);
```

Reply:
updated, thanks